src/models/Block.ts

```diff
@@ -0,0 +1,14 @@
+export interface IBlock {
+  _id: string;
+  height: number;
```

Comment:
Ok, i'll convert them, but sanctuary API returns numbers, so big integers will be lost anyway and converting `number` to `BigNumber` in the SDK would not give any additional safety. Maybe we should store heights in the mongo not as numbers, but as strings? And also return them as strings? Otherwise, safety of big integers are lost by the time of storing them in mongo.
@noahmatisoff  @DZariusz

Reply:
Also, do we need to use BigNumber for leaf values?

Reply:
in that case we need to change that in BE, we actually have old task for this `number` issue, so I think its time to start addressing it. 

Since this SDK will be used to simply read data, let's use `string` for numeric values that cames from blockchain. 

> Also, do we need to use BigNumber for leaf values?

leaf value can be anything, not only number. Lets use `sting`.

Reply:
@DZariusz  Ok, i'll look at that issue.

> leaf value can be anything, not only number. Lets use sting

Actually that's not what meant. I mean do we need to convert strings to BigNumber if the leaf type is LeafType.INTEGER or LeafType.FLOAT? Because right now i'm just doin parseInt or parseFloat

Reply:
> @DZariusz Ok, i'll look at that issue.

the issue is separate problem, lets focus on SDK first and ignore fact that sanctuary stores it as number.

lets do not convert to BigNumber. I left separate comment for `resolveLeafValue`.